app/models/insights_client_report_status.rb

```diff
@@ -0,0 +1,60 @@
+class InsightsClientReportStatus < HostStatus::Status
+  REPORT_INTERVAL = 48.hours
+
+  REPORTING             = 0 # insights-client registered & reporting
+  NO_REPORT             = 1 # insights-client registered but not getting data
+  NOT_MANAGED           = 2 # insights-client should not be registered
+  NOT_MANAGED_WITH_DATA = 3 # insights-client should not be registered but getting data
+
+  def self.status_name
+    N_('Insights')
+  end
+
+  def to_label(_options = {})
+    case status
+      when REPORTING
+        N_('Reporting')
+      when NO_REPORT
+        N_('Not reporting')
+      when NOT_MANAGED
+        N_('Not reporting')
+      when NOT_MANAGED_WITH_DATA
+        N_('Reporting')
+    end
+  end
+
+  def to_global(_options = {})
+    case status
+      when REPORTING
+        ::HostStatus::Global::OK
+      when NO_REPORT
+        ::HostStatus::Global::ERROR
+      when NOT_MANAGED
+        ::HostStatus::Global::OK
+      when NOT_MANAGED_WITH_DATA
+        ::HostStatus::Global::WARN
```

Comment:
I was thinking about this but always come back to the conclusion, warning is probably the right value. While users may deploy insights-client through other means (ansible role, puppet module, manually) the parameter says it should not be deployed so users should either change the param or be warned about that. So :+1: